### PR TITLE
Grill Comment Integration

### PR DESCRIPTION
**Pull request by teodorus-nathaniel**
# Before Merge
- [x] Change the hubId to use the real space for comments

### CODE REVIEW

File: .github/workflows/build-deploy.yml [hunk 1]
```diff
@@ -72,6 +72,7 @@ jobs:
             GH_GA_ID=UA-204866512-2
             GH_APP_KIND=polkaverse
             GH_HCAPTCHA_SITE_KEY=${{ secrets.PROD_HCAPTCHA_SITE_KEY }}
+            GH_AMP_ID=d9f7a97f3cdc1eb7b4298af4f17c202b
```
Comment (olehmell) [inline .github/workflows/build-deploy.yml:75]:
```suggestion
            GH_AMP_ID=2eeca0e8a0163c89e3f023c971e426a6
```

File: .github/workflows/feature-based.yaml [hunk 2]
```diff
@@ -62,6 +62,7 @@ jobs:
             GH_GA_ID=fake
             GH_APP_KIND=polkaverse
             GH_HCAPTCHA_SITE_KEY=3beeddac-2dce-41cc-8e18-338118426c38
+            GH_AMP_ID=a4dc29d356f19fd9788f24b493b462bf
```
Comment (olehmell) [inline .github/workflows/feature-based.yaml:65]:
```suggestion
            GH_AMP_ID=71bf5a46800fedba5e9a01243b988164
```

File: src/components/chat/ChatIframe.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import { useSendEvent } from 'src/providers/AnalyticContext'
+import { useSetChatTotalMessageCount } from 'src/rtk/app/hooks'
+import { useAppSelector } from 'src/rtk/app/store'
+import { ChatEntity } from 'src/rtk/features/chat/chatSlice'
+
+export type ChatIframeProps = ComponentProps<'div'> & {
+  onUnreadCountChange?: (count: number) => void
+}
+
+const COMMENTS_HUB_ID = '1032'
```
Comment (olehmell) [inline src/components/chat/ChatIframe.tsx:16]: Extract it to config

File: src/components/chat/ChatIframe.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    const body = summarize(post.content?.body ?? '', { limit: 50 })
+    return {
+      hub: {
+        id: COMMENTS_HUB_ID,
+      },
+      theme: 'light',
+      rootFontSize: '1rem',
+      channel: {
+        type: 'resource',
+        resource: new Resource({
```
Comment (olehmell) [inline src/components/chat/ChatIframe.tsx:71]: I asked Alex about Resource type, maybe it should be "subsocial" chain instead polkaverse app

**Review by olehmell — CHANGES_REQUESTED** [review]

File: src/components/chat/ChatIframe.tsx (hunk 3 above)
Comment (teodorus-nathaniel) [reply src/components/chat/ChatIframe.tsx]: Updated, moved it into
`common` config with name `commentsHubId`

**Pull request merged**